### DISCUSSION TITLE
Hotspot maps

### DISCUSSION THREAD

**Discussion by Goldengame04** [janstehno/cotw-companion#18] [Ideas]
I had an idea for all animal in the section maybe a a Hotspots maps

**Comment by janstehno** [comment]
I would implement this feature into already created maps rather than to each animal. But it should be possible, I will look into it.

**Reply by JeanRoxus** [reply]
I feel that the need zones on some maps are not updated completely

**Reply by janstehno** [reply]
What do you mean exactly? The hotspots feaature is not implemented yet.
